### ISSUE TITLE
Side nav question

### ISSUE THREAD

**Issue by davidstokes5000** [almasaeed2010/AdminLTE#930]
Hi, Just want to say it's a great design and I love it. 

I'm only starting to use it and have a question. 

The side navigation. Is there a way to only have to update it once and it will appear on all pages. It's hard work making 1 change to the nav and then having to update it across 20+ pages each time. 

thanks.

**Comment by superjija** [comment]
Hi David,
Can you use PHP? If you can, make separated files for menus, header, footer, content.. and use include() for joining this parts.

**Comment by davidstokes5000** [comment]
I don't.. but the coder I'll be hiring to get the admin panel functional will. So that will be a great idea. 

Thanks.

**Comment by janhaa** [comment]
I would advise to use some kind of MVC for that, AngularJS makes what you want pretty easy.

**Comment by davidstokes5000** [comment]
I don't know what MVC is.. and i don't know JS :( .. however I was able to get the php include working on my local server using this
`<?php
include 'meta.php';
include 'header.php';
include 'nav.php';?>`

and then I made pages called header / nav / mete and cut the code into them. It's working. I'm not finished yet I need to get footer added as well. i'm still going through the code. 

What my plan is is to have each page I made call these so each page will have those include commands and then the normal content I want on that page in the middle. 

I'm sure there there are easier ways.. but I'm not a coder.. I know HTML and CSS..

Thanks.
